tests/conftest.py

```diff
@@ -29,11 +30,37 @@ def test_app():
             }
         )
     )
-    app.oauth = OauthHelper(app.client_id, app.client_secret, "")
+    app.oauth = OauthHelper(app.client_id, app.client_secret, app.callback_url)
     yield app
     loop.run_until_complete(app.destroy_app())
 
 
+@pytest.yield_fixture(scope="session")
```

Comment:
These are very useful. Consider upstreaming these fixtures to the test utils repo

Reply:
As this is blocking, we suggest creating a ticket for this refactor?

Reply:
https://nhsd-jira.digital.nhs.uk/browse/APM-2029